docs/source/example_vmec.rst

```diff
@@ -0,0 +1,318 @@
+ 1200  6.41E-03  1.22E-05  6.86E-05  2.773E-01  9.00E-01  1.3019E-02
+ 1400  6.03E-03  1.20E-05  7.04E-05  2.773E-01  9.00E-01  1.3019E-02
+ 1600  8.37E-03  1.70E-05  9.62E-05  2.772E-01  9.00E-01  1.3019E-02
+ 1800  9.33E-03  1.88E-05  1.05E-04  2.772E-01  9.00E-01  1.3019E-02
+ 2000  9.96E-03  2.01E-05  1.10E-04  2.772E-01  9.00E-01  1.3019E-02
+
+  NS =  100 NO. FOURIER MODES =  116 FTOLV =  1.000E-12 NITER =   4000
+  PROCESSOR COUNT - RADIAL:    1
+
+  ITER    FSQR      FSQZ      FSQL    RAX(v=0)    DELT       WMHD
+
+    1  3.93E-01  1.50E-01  2.53E-04  2.772E-01  9.00E-01  1.3018E-02
+  200  9.79E-07  5.70E-08  9.84E-10  2.774E-01  6.87E-01  1.3018E-02
+  400  3.58E-08  2.76E-09  4.35E-11  2.772E-01  6.87E-01  1.3018E-02
+  600  5.12E-09  4.91E-10  3.09E-12  2.772E-01  6.87E-01  1.3018E-02
+  800  1.27E-09  1.36E-10  4.77E-13  2.772E-01  6.87E-01  1.3018E-02
+ 1000  4.85E-10  2.98E-11  2.01E-13  2.772E-01  6.87E-01  1.3018E-02
+ 1200  3.55E-10  1.25E-11  8.46E-14  2.772E-01  6.87E-01  1.3018E-02
+ 1400  3.50E-10  1.01E-11  7.16E-14  2.772E-01  6.87E-01  1.3018E-02
+ 1600  3.28E-10  8.89E-12  6.47E-14  2.772E-01  6.87E-01  1.3018E-02
+ 1800  2.93E-10  7.27E-12  5.59E-14  2.772E-01  6.87E-01  1.3018E-02
+ 2000  2.67E-10  6.34E-12  5.10E-14  2.771E-01  6.87E-01  1.3018E-02
+ 2200  2.52E-10  5.93E-12  4.96E-14  2.771E-01  6.87E-01  1.3018E-02
+ 2400  2.45E-10  5.73E-12  4.77E-14  2.771E-01  6.87E-01  1.3018E-02
+ 2600  2.45E-10  5.67E-12  4.76E-14  2.771E-01  6.87E-01  1.3018E-02
+ 2800  2.50E-10  5.72E-12  4.85E-14  2.771E-01  6.87E-01  1.3018E-02
+ 3000  2.59E-10  5.88E-12  5.01E-14  2.771E-01  6.87E-01  1.3018E-02
+ 3200  2.74E-10  6.17E-12  5.25E-14  2.771E-01  6.87E-01  1.3018E-02
+ 3400  2.95E-10  6.57E-12  5.58E-14  2.771E-01  6.87E-01  1.3018E-02
+ 3600  3.21E-10  7.09E-12  6.01E-14  2.771E-01  6.87E-01  1.3018E-02
+ 3800  3.54E-10  7.73E-12  6.52E-14  2.771E-01  6.87E-01  1.3018E-02
+ 4000  3.93E-10  8.49E-12  7.12E-14  2.770E-01  6.87E-01  1.3018E-02
+
+    simsopt._core.util.ObjectiveFailure: VMEC did not converge. ierr=2
+
+A ``RuntimeError`` is raised if VMEC fails, and the ``ierr`` code indicates the type of error. 
+The interpretation of the error code is listed below:
+
+- norm_term_flag=0: Normal termination
+- bad_jacobian_flag=1: Initial guess for the coordinate transformation is bad. 
+- more_iter_flag=2: More iterations are required
+- jac75_flag=4: More than 75 Jacobain iterations (decrease ``delt``)
+- input_error_flag=5: Input file is not properly formatted 
+- phiedge_error_flag=7: ``phiedge`` has the wrong sign in vacuum subroutine (only for free boundary)
+- ns_error_flag=8: ``ns_array`` must not all be zeros 
+- misc_error_flag=9: Error reading ``mgrid`` file 
+- successful_term_flag=11: Normal termination
+- bsub_bad_js1_flag=12: bsubu or bsubv js=1 component non-zero 
+- r01_bad_value_flag=13: rmnc(0,1) is zero 
+- arz_bad_value_flag=14: anorm or aznorm equal zero in bcovar 
```

Comment:
it would be great to understand in more detail what exactly these errors actually mean.  

For example, I don't know what `anorm/aznorm equal to zero in bcovar` actually means.  

What does it mean that `Initial guess for the coordinate transformation is bad.`?

Is there a resource we can be pointed to that has these technical terms explained?